templates/web-application/package.json

```diff
@@ -4,20 +4,28 @@
   "description": "",
   "type": "module",
   "scripts": {
+    "postinstall": "pnpm dlx rimraf node_modules/.cache",
+
     "dev": "webpack serve --config webpack.dev.js",
     "dev-msw": "webpack serve --config webpack.dev.js --env use-msw",
     "build": "webpack --config webpack.build.js",
-    "test": "jest",
-    "serve-build": "pnpm http-server dist -p 8080 -P http://localhost:8080?",
-    "clean": "pnpm dlx rimraf dist node_modules/.cache",
-    "reset": "pnpm run clean && pnpm run reset:modules",
-    "reset:modules": "pnpm dlx rimraf node_modules pnpm-lock.yaml",
+
     "lint": "pnpm run \"/^lint:.*/\"",
     "lint:eslint": "eslint --ext .js,.ts,.tsx . --max-warnings=-1 --cache --cache-location node_modules/.cache/eslint",
     "lint:stylelint": "stylelint \"**/*.css\" --max-warnings 0 --cache --cache-location node_modules/.cache/stylelint",
     "lint:types": "pnpm exec tsc",
+
+    "clean": "pnpm dlx rimraf dist node_modules/.cache",
+    "reset": "pnpm run clean && pnpm run reset:modules",
+    "reset:modules": "pnpm dlx rimraf node_modules pnpm-lock.yaml",
+
     "analyze": "webpack --config webpack.build.js --analyze",
-    "postinstall": "pnpm dlx rimraf node_modules/.cache"
+    "serve-build": "pnpm http-server dist -p 8080 -P http://localhost:8080?",
+
+    "test": "jest",
+
+    "storybook": "storybook dev -p 6006",
+    "chromatic": "pnpm dlx chromatic --only-changed"
```

Comment:
And I was wondering, what about Turbocache?

Reply:
I was thinking about that yesterday too, having `pnpm dlx` in scripts require having `pnpm` installed to use the template.

For example, the dependencies installation for the template will fail if we don't have `pnpm` installed because of the `postinstall`.

Using `rimraf` as a dependencies, like metion above for `chromatic` can fix that.

Reply:
The issue with having `rimraf` as a dependency is that you must have the package, installed. For a `postinstall` script it's a great solution, for a `reset` script it's not ideal.

Still, you'll need to have PNPM installed anyway for most of our templates that will be monorepos (because of PNPM workspaces feature) so it's not a big deal for me to have this prerequisite.

My biggest concern in fact would be how to make it obvious that it will not work if PNPM is not installed? I don't want developers less experienced with the frontend ecosystem to spend hours trying to debug an issue that is caused by PNPM not being installed.

Reply:
With `NPM` installed by default, can it be a good idea to run `npx` instead of `pnpm dlx` for `postinstall` specifically?

Reply:
That would work but only for poly-repo, not for monorepo.

For a monorepo the `postinstall` script is the following: `"postinstall": "pnpm -r --parallel --include-workspace-root exec pnpm dlx rimraf node_modules/.cache",`

Reply:
I'm not 100% sure of how people are going to use chromatic yet. If people are not running local chromatic test often, we might as well not install chromatic for everyone everytime. 

I think the usage will come from the pipeline, and depending on the platform used for the workflows, they probably won't use this command since they have built in stuff. 

i'm open to change this if you think we should add a peer dependency

Reply:
Got it, I haven't thought about it this way. Seems fine to me as long as we don't need things like the `isChromatic` (its exported by the chromatic package) utility in our codebases.